src/main/java/org/mockito/internal/creation/bytebuddy/ByteBuddyMockMaker.java

```diff
@@ -37,8 +36,8 @@ public ByteBuddyMockMaker() {
         } catch (ClassCastException cce) {
             throw new MockitoException(join(
                     "ClassCastException occurred while creating the mockito mock :",
-                    "  class to mock : " + describeClass(mockedProxyType),
-                    "  created class : " + describeClass(settings.getTypeToMock()),
+                    "  class to mock : " + describeClass(settings.getTypeToMock()),
+                    "  created class : " + describeClass(mockedProxyType),
```

Comment:
It seems that these lines are untested. Are we able to come up with a test for this? E.g. why would we reorder these values.